src/components/Brand/Brand.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, DimensionValue, Text, Image } from 'react-native';
+
+type Props = {
+  height?: DimensionValue;
+  width?: DimensionValue;
+  mode?: 'contain' | 'cover' | 'stretch' | 'repeat' | 'center';
```

Comment:
could you try typing it as `ImageProps['resizeMode']` ? Or some type directly from the RN library instead of custom defining it